src/components/Teaser/Teaser.scss

```diff
@@ -2,10 +2,22 @@
     min-width: 0;
     margin: 0;
     display: flex;
+    flex: 1;
     flex-direction: column;
+    height: 63px;
 
-    &__titleLine {
+    &__line {
         display: flex;
+        flex: 1;
+        align-items: center;
+        padding-bottom: var(--spacing-2x);
+    }
+
+    &__lastLine {
+        display: flex;
+        flex: 1;
+        align-items: center;
+        justify-content: flex-end;
```

Comment:
I'd suggest to use `:last-child` pseudo-selector here instead of introducing new element `lastLine` since it exists especially for this kind of cases
```css
    &__line:last-child {
        padding-bottom: 0;
        justify-content: flex-end;
    }
```

Reply:
agree, looks better!